types/tests/augmentation-test.ts

```diff
@@ -0,0 +1,11 @@
+import Banana from '../index';
+
+declare module '../banana-i18n' {
+	type A = 'test';
```

Comment:
Can we use something else than 'test' since it is used as message key as well. More better if we use more explicit message key name. Just to avoid confusions

Reply:
Done